### ISSUE TITLE
(Item request) Bigger wind turbines

### ISSUE THREAD

**Issue by pooing1234** [Electrical-Age/ElectricalAge#666]
I think it would be good to have another turbine, but bigger, and taller. It could produce up to 2, or 4 KW's peak.

**Comment by AMIDIBOSS** [comment]
2kW peak? Little overkill, but still, good idea to have bigger ones

**Comment by pooing1234** [comment]
@AMIDIBOSS Yeah, may be a little overkill, just not used to the ELA's system.

**Comment by Baughn** [comment]
It's something we'd need to give careful thought.

One rule I've set myself is that no power producer should be universally superior or interior to another. Strengths should always be balanced by weaknesses.

- Solar panels only work during daytime, don't work very well if it's raining, and batteries aren't good enough to run very much machinery off of. They're good to reduce fuel usage, but you can't run your base off just the panels.

- Heat turbines are difficult to make efficient, and don't produce very much energy overall.

- Gas turbines need to be spun up, and unless you're careful they'll easily end up overshooting and exploding.

- Steam turbines absolutely require some way to prevent overshoot... They're very efficient, but react too slowly to deal with intermittent load very well.

- Wind turbines unpredictably fall to zero output, and produce very little to begin with...

And water turbines should probably be removed.

Adding larger wind turbines would make it easier to spam them, which is something I absolutely do not want to make a possibility.

**Comment by AMIDIBOSS** [comment]
And if wind turbines output Mechanical power?

**Comment by Baughn** [comment]
Then I'll need to finally implement those gearboxes. :-)

**Comment by pooing1234** [comment]
@AMIDIBOSS You could make some of the fans mechanical as well.

**Comment by OmegaHaxors** [comment]
Maybe rebrand the water turbines as shaft-powered to give them a purpose; the challenge would come from trying to get them all to spin at the same speed without causing issues in the shafts. Sure you _could_ spam them but GL trying to get any more than a handful to agree on anything.

**Comment by Baughn** [comment]
Mm.

When it comes to shaft power, the turbines are easy-mode; efficiency aside, they just spin up no matter what. The water/wind turbine equivalents would hit a max speed and then eventually start *dropping* power... the water turbine would be trying to speed up the river if driven too fast...

A good water/wind expansion would depend on that, but I'm still not sure of the details. For water, I'd like to implement peltier turbines at some point.
